File: Blablacar/Blablacar.Infrastructure.Data/Entities/Trip.cs
using System;

namespace Blablacar.Infrastructure.Data
{
    public class Trip : BaseObject
    {
        public string From { get; set; }

        public string To { get; set; }

        public DateTime DepartureTime { get; set; }

        public Driver Driver { get; set; }

        public Customer Customer { get; set; }

        public int Price { get; set; }
    }
}

File: BlablacarWebFrontEnd/Blablacar/src/models/trip.ts
import { IDriver } from './driver';
import { ICustomer } from './customer';
import { IBaseModel } from './base-model';

export interface ITrip extends IBaseModel
{
    from: string;
    to: string;
    departureTime: string;
    driver: IDriver;
    customer: ICustomer;
    price: number
}